### PR TITLE
#2512 update route constants

### LINKED ISSUE #2512: Update route constants
## Is your feature request related to a problem? Please describe.

Placeholder issue for updating `getColumns` with route constants.

## Describe the solution you'd like

See above.

## Implementation

Update navigation constants, add to `getColumns`.

## Describe alternatives you've considered

N/A.

## Additional context

N/A.

### PR DESCRIPTION
**Pull request by wlthomson**
Fixes #2512.

## Change summary

Adds remaining `getColumn` constants.

## Testing

TODO.

### Related areas to think about

N/A.

### CODE REVIEW

File: src/navigation/constants.js [hunk 1]
```diff
@@ -25,6 +25,7 @@ export const ROUTES = {
   CUSTOMER_INVOICES: 'customerInvoices',
 
   SUPPLIER_INVOICE: 'supplierInvoice',
+  SUPPLIER_INVOICE_WITH_PRICES: 'supplierInvoiceWithPrices',
```
Comment (mark-prins) [inline src/navigation/constants.js]: This isn't used anywhere.. should the `SupplierInvoicePage` be updated to use the CONSTANT?
Or is that part of another issue?

Comment (wlthomson) [reply src/navigation/constants.js]: Good idea. 554d700c9f416409d6c90af176f5a550a05f418a.

Comment (wlthomson) [reply src/navigation/constants.js]: Good idea. 554d700c9f416409d6c90af176f5a550a05f418a.

**Comment by Rebecca367** [comment]
Is there any testing needs to be done for this issue?TODO doesn't make sense to me, sorry
!

**Comment by wlthomson** [comment]
@Rebecca367 No logic changes here, so all good to mark as tested IMO.

**Comment by Rebecca367** [comment]
@wlthomson 
cool..move to done then.

**Pull request merged**